assets/locales/service.en.toml

```diff
@@ -199,3 +199,20 @@ one = "Cancelled"
 [ReleasePageType]
 description = "Release"
 one = "Release"
+
+[NoReleasesFound]
+description = "No releases found"
+one = "No releases found"
+
+[NoReleasesFoundDescriptionPublished]
```

Comment:
Corresponding entries for Published, Upcoming, and Cancelled must be added to the Welsh locale otherwise the frontend will crash when it tries to look them up.